src/browser/modules/Stream/CypherFrame/helpers.js

```diff
@@ -74,10 +72,15 @@ export function getBodyAndStatusBarMessages (result, maxRows) {
     streamMessage = streamMessage[0].toUpperCase() + streamMessage.slice(1)
   }
 
+  const systemUpdatesValue = get(result, 'summary.counters._systemUpdates')
```

Comment:
I have had to pull it here similar to another section in the code, no ideal but only way I can see to get this value (suggestions welcome).

I haven't defaulted to anything as if it's undefined thats fine, and the logic should still hold...

in terms of naming, `systemUpdatesValue` or `systemUpdatesCount`?

Reply:
I think this is the only place where it's available after serialization.